.github/workflows/system-tests.yml

```diff
@@ -84,8 +83,7 @@ jobs:
         working-directory: ./solidity
         run: |
           npx hardhat node --no-deploy &
-          HH_NODE_PID=$!
-          while ! lsof -n -Fn -p $HH_NODE_PID | grep -q '^n.*:8545$'; do sleep 10; done
+          while [[ -z $(lsof -i :8545 -t) ]]; do echo "Waiting for port 8545 to be open..."; sleep 10; done
```

Comment:
The same comment as [here](https://github.com/keep-network/tbtc-v2/pull/719/files#r1374357224).